docker-storage-setup.sh

```diff
@@ -215,11 +215,11 @@ get_devicemapper_config_options() {
   done )
 
   if extra_options_has_dm_fs; then
-    # dm.fs option defined in ${EXTRA_DOCKER_STORAGE_OPTIONS}
+    # dm.fs option defined in ${EXTRA_STORAGE_OPTIONS}
     dm_fs=""
   fi
 
-  storage_options="DOCKER_STORAGE_OPTIONS=\"--storage-driver devicemapper ${dm_fs} --storage-opt dm.thinpooldev=$POOL_DEVICE_PATH $(get_deferred_removal_string) $(get_deferred_deletion_string) ${EXTRA_DOCKER_STORAGE_OPTIONS}\""
+  storage_options="STORAGE_OPTIONS=\"--storage-driver devicemapper ${dm_fs} --storage-opt dm.thinpooldev=$POOL_DEVICE_PATH $(get_deferred_removal_string) $(get_deferred_deletion_string) ${EXTRA_STORAGE_OPTIONS}\""
```

Comment:
Then we should just use the transition name like we did at the bottom.  If user specifies DOCKER_STORAGE_OPTIONS we want and assign to STORAGE_OPTIONS.